README.md

```diff
@@ -593,6 +594,15 @@ The task consists in predicting the next word given the previous ones.
 The corpus contains approximately 30 million sentences of an average length of about 25 words. 
 In total, there are about 800 thousand (unique) words in the vocabulary, which makes it a very memory intensive problem.
 
+<a name='dl.loadTwitterSentiment'></a>
+## loadTwitterSentiment
+
+```lua
+train, valid, test, vocab, ivocab, unigram = dl.loadTwitterSentiment(
```

Comment:
Can you append the last 3 returned values to the first 3. So then we can do set.vocab, set.ivocab and set.unigram, where set is train, valid, and test. Standardizes it with the other loaders and returns less arguments.